tests/main/views/test_users.py

```diff
@@ -632,20 +605,19 @@ def test_new_password_is_not_audited(self):
                         'password': 'not-in-my-audit-event'}}),
                 content_type='application/json')
 
-            assert_equal(response.status_code, 200)
+            assert response.status_code == 200
             data = json.loads(response.get_data())['users']
-            assert_equal(data['emailAddress'], 'test@digital.gov.uk')
+            assert data['emailAddress'] == 'test@digital.gov.uk'
 
             audit_response = self.client.get('/audit-events')
-            assert_equal(audit_response.status_code, 200)
+            assert audit_response.status_code == 200
             data = json.loads(audit_response.get_data())
 
-            assert_equal(len(data['auditEvents']), 1)
-            assert_equal(data['auditEvents'][0]['type'], 'update_user')
-            assert_equal(
-                data['auditEvents'][0]['data']['update']['password'],
-                'updated'
-            )
+            assert len(data['auditEvents']) == 1
+            assert data['auditEvents'][0]['type'] == 'update_user'
+            assert (
+                data['auditEvents'][0]['data']['update']['password'] ==
+                'updated')
```

Comment:
Yeah, this is a weird formatting thing that comes out of `nose2pytest`.  I cleaned up most of them but not all of them.